### ISSUE TITLE
Odd behaviour when trying to find current working directory using git status

### ISSUE THREAD

**Issue by floriank** [divmain/GitSavvy#718]
Hey,

I have recently done a fresh install of GitSavvy and noticed a strange behaviour change from my previous installation (work laptop).

Assuming the following project setup `test-project.sublime-project`:

```json
{
	"folders":
	[

		{
			"path": "frontend",
			"name": "A generic frontend"
		},
		{
			"path": "api",
			"name": "An API"
		},
		{
			"path": "gateway",
			"name": "A gateway"
		},
		{
			"path": "service",
			"name": "A service"
		}
	]
}
```

_Note_: All of these folders are initialized as git repos separately.

Using `git status` from an arbitrary file in one of these directories will always show the git status for the project `frontend`.

I did some digging in the code and came across https://github.com/divmain/GitSavvy/blob/c2d42fb8b7799a109cc00ad35215f87ae9668326/core/git_command.py#L215, which recently got refactored in #712. 

`find_working_dir` will _always_ prefer returning the first dir in `open_folders`, which explains the behaviour to me, but I gotta ask: 

Was this intended? It's a huge bummer to me when it comes to UX, as i expect `git status` to open the status page for the git repo I am currently in, i.e. the file I have currently open. Additionally, I cannot switch quickly between the individual repos anymore, which I greatly enjoyed beforehand.

Anyway, if this was intended, please close the issue.

**Comment by randy3k** [comment]
There was an issue for editing out of repo files, that why it got refactored. We should definitely support multi directory. Thanks for opening the issue.

**Comment by randy3k** [comment]
it should be fixed in #719.

**Comment by floriank** [comment]
Thank you!
